scripts/backup

```diff
@@ -39,6 +41,12 @@ ynh_backup "$final_path"
 
 ynh_backup "/etc/nginx/conf.d/$domain.d/$app.conf"
 
+#=================================================
+# BACKUP THE POSTGRESQL DATABASE
+#=================================================
+
+ynh_psql_dump_db --database="$db_name" > ${YNH_CWD}/dump.sql
```

Comment:
why do you use ${YNH_CWD}?
it's not part of https://github.com/YunoHost/example_ynh/blob/master/scripts/backup#L70
never saw it before

Reply:
Took it from here: https://github.com/YunoHost-Apps/synapse_ynh/blob/57a79f89c59fcb29225a7d3f10fcdd8e6d1a4f68/scripts/restore#L116

Reply:
Also in https://yunohost.org/#/packaging_apps_scripts_fr

Reply:
ok, interesting